troposphere/ecr.py

```diff
@@ -20,4 +20,5 @@ class Repository(AWSObject):
         'LifecyclePolicy': (LifecyclePolicy, False),
         'RepositoryName': (basestring, False),
         'RepositoryPolicyText': (policytypes, False),
+        'Tags': ((Tags, list), False),
```

Comment:
When adding new Tags there is a preference not to include 'list' which is being deprecated. Can you change this to just be 'Tags'? Likely need to tweak the test as well.

Reply:
Updated!